### PR TITLE
Vdm 45 and 60

### PR DESCRIPTION
**Pull request by manjujames**

### CODE REVIEW

File: lib/python/vdm/server/HTTPListener.py [hunk 1]
```diff
@@ -2274,7 +2270,20 @@ def get(database_id, server_id):
             else:
 
                 try:
-                    client = voltdbclient.FastSerializer(str(server[0]['hostname']), 21212)
+                    if not server[0]['client-listener']:
```
Comment (akhanzode) [inline lib/python/vdm/server/HTTPListener.py]: should check for length as well right?

Comment (manjujames) [inline lib/python/vdm/server/HTTPListener.py]: not actually checks that.

**Comment by manjujames** [comment]
squash merged.

File: lib/python/vdm/server/HTTPListener.py [hunk 2]
```diff
@@ -2274,7 +2270,20 @@ def get(database_id, server_id):
...
+                        client_port = 21212
+                        client_host = str(server[0]['hostname'])
+                    else:
+                        client_listener = server[0]['client-listener']
+                        if ":" in client_listener:
```
Comment (pdshaw) [inline lib/python/vdm/server/HTTPListener.py]: urlparse python module will do this. not a big deal, just fyi

**Pull request closed without merging**